exporters/datadog/opentelemetry-exporters-datadog.gemspec

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+# require 'opentelemetry-exporters-datadog/exporters/datadog/version'
+require 'opentelemetry/exporters/datadog/version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-exporters-datadog'
+  spec.version     = OpenTelemetry::Exporters::Datadog::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'Datadog trace exporter for the OpenTelemetry framework'
+  spec.description = 'Datadog trace exporter for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = ::Dir.glob('lib/**/*.rb') +
+               ::Dir.glob('*.md') +
+               ['LICENSE', '.yardopts']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 2.5.0'
+
+  spec.add_dependency 'ddtrace'
```

Comment:
You would probably have a minimum version of `ddtrace` supported here.

I suggest taking at look at the modules and classes you reference in this PR and find when they were introduced with a suitable implementation in `ddtrace`.

Reply:
So the main thing is the `Datadog::Writer` module, which it has just occurred to me may get deprecated/removed in future? Would it make more sense to just move to using the `AsyncWriter` now and making `0.35.0` a min version?